arangod/Aql/Query.h

```diff
@@ -141,7 +141,7 @@ class Query : public QueryContext, public std::enable_shared_from_this<Query> {
   QueryResult parse();
 
   /// @brief explain an AQL query
-  QueryResult explain();
+  QueryResult explain(bool optimize = true);
```

Comment:
I would prefer to not use defaulted parameters, but rather have callers specify the parameter value explicitly.
```suggestion
  QueryResult explain(bool optimize);
```